### PR TITLE
[WIP] Converting tables in Autoyast guide into variable lists

### PR DESCRIPTION
**Pull request by lproven**
Replace table in "System Registration & Extension Selection" § with variablelist

### Description

The tables in the AutoYast guide do not scale well in some output formats and are tricky to maintain. (For instance, they require manual insertion of line-breaks in [screen] statements.

This is the first of a series of planned replacements of tables with simpler variablelist structures, whose formatting should scale more easily.

### Are there any relevant issues/feature requests?

See Trello card:
* https://trello.com/c/zMJy4Uqz/151-restructure-autoyast-guide-fix-wide-tables

### To which product versions do the changes apply?

The first column is to be filled by the requester, the second column is to be filled by the doc team after the changes have been backported to the maintenance branches.

| Code 15     | Backport Done
| ----------  | ---------- 
| [x] 15 SP3  | (`master` only)   
| [x] 15 SP2  | [ ] 
| [x] 15 SP1  | [ ] 
| [x] 15 SP0  | [ ] 

| Code 12     | Backport Done
| ----------  | ---------- 
| [ ] 12 SP5  | [ ] 
| [ ] 12 SP4  | [ ] 
| [ ] 12 SP3  | [ ] 
| [ ] 12 SP2  | [ ]

### CODE REVIEW

File: xml/ay_bootloader.xml [hunk 1]
```diff
@@ -102,352 +102,284 @@
...
+      </listitem>
+     </varlistentry>
+     
+     <varlistentry>
+      <term>gfxmode</term>
+      <listitem>
+       <para>
+        Graphical resolution of the &grub; screen (requires |&lt;terminal&gt; to
+        be set to <literal>gfxterm</literal>.
+       </para>
```
Comment (cjschroder) [inline xml/ay_bootloader.xml:208]: Looks like typos here, unclosed parenthese, and a pipe

File: xml/ay_bootloader.xml [hunk 2]
```diff
@@ -102,352 +102,284 @@
...
+       <para>
+        Valid entries are <literal>auto</literal>, <literal><replaceable>
+        HORIZONTAL</replaceable>x<replaceable>VERTICAL</replaceable></literal>,
+        or <literal><replaceable>HORIZONTAL</replaceable>x<replaceable>VERTICAL
+        </replaceable>x<replaceable>COLOR DEPTH</replaceable></literal>. You can
+        see the screen resolutions supported by &grub; on a particular system by
+        using the <command>vbeinfo</command> command at the &grub; command line
+        in the running system.
```
Comment (cjschroder) [inline xml/ay_bootloader.xml:216]: vbeinfo is not in suse. xrandr works.

**Review by cjschroder — CHANGES_REQUESTED** [review]
This looks waaaay better!

File: xml/ay_bootloader.xml (hunk 2 above)
Comment (lproven) [reply xml/ay_bootloader.xml:216]: > xrandr works.

Snag is, that requires X.11, I think.

But this is a good catch.

From the shell, there is `hwinfo --vbe` but I think this is a GRUB-EFI issue.

I found a suggestion of `videoinfo` but I do not have a UEFI machine to test. Do you?

Comment (cjschroder) [reply xml/ay_bootloader.xml:216]: Oh never mind, "using the vbeinfo command at the GRUB 2 command line in the running system." Ignore.

**Review by cjschroder — APPROVED** [review]

**Review by cjschroder — APPROVED** [review]
This is so much better! It builds, and it looks great.

**Comment by lproven** [comment]
> This is so much better! It builds, and it looks great.

Glad to hear it.

I _think_ that is all the 2-column tables converted now. Have I missed any? 

I fear that some of the 3-column tables will get more confused in variable-list format, so I propose leaving them for now. I am not sure what resolution is best:
* hierarchical itemised lists (for some);
* variable lists but with a rigorous paragraph structure, e.g. term (type), para 1 = explanation, para 2 = example;
* possibly imposing a more rigorous structure on the existing varlists, to comply with the above proposal;
* something else, including just leaving them. 

Some of the tables have 1-to-1 matches between columns (e.g. changed firewall settings) and should stay, but others do not, IMHO.

Debate here or elsewhere?

**Comment by cjschroder** [comment]
This has gotten so big IMO it is a good time to merge, then create a new branch for any further corrections. These 2-column tables remain, though I think they are fine as-is:
4.35 DNS Server
TABLE C.1: ADVANCED linuxrc KEYWORDS
D.2.3 Renamed Software Patterns
TABLE D.1: AUTOYAST FIREWALL CONFIGURATION IN SLE 15: BACKWARD COMPATIBILITY
TABLE D.2: MAPPING OF SUSEFIREWALL2 AND firewalld ZONES

Some of the 3-column tables really need to be changed, but not all of them. Tomorrow I shall look at all of them.




The three-column tables look pretty awkward in PDF, but some are OK. I will look at these tomorrow.

**Pull request merged**